sapi/apache2handler/php_functions.stub.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/** @return object|false */
+function apache_lookup_uri(string $filename) {}
+
+function virtual(string $uri): bool {}
+
+function apache_request_headers(): array {}
+
+function getallheaders(): array {}
+
+function apache_response_headers(): array {}
+
+/** @return string|false */
+function apache_note(string $note_name, string $note_value = "") {}
```

Comment:
I think we can now declare union return types as well:

```suggestion
function apache_note(string $note_name, string $note_value = ""): string|false {}
```